src/lib/gssapi/krb5/gssapi_krb5.h

```diff
@@ -96,6 +96,15 @@ GSS_DLLIMP extern const gss_OID_desc krb5_gss_oid_array[];
  */
 GSS_DLLIMP extern const gss_OID GSS_KRB5_CRED_NO_CI_FLAGS_X;
 
+/*
+ * This OID can be used with gss_inquire_cred_by_oid(0 to retrieve the
+ * impersonator name (if any).
+ *
+ * iso(1) member-body(2) United States(840) mit(113554) infosys(1) gssapi(2)
+ * krb5(2) krb5_gssapi_ext(5) get_cred_impersonator(14)
+ */
+GSS_DLLIMP extern const gss_OID GSS_KRB5_GET_CRED_IMPERSONATOR;
```

Comment:
Now that I see the expansion of this OID, it's in an arc I haven't familiarized myself with, and which I don't see a registry for.  How did you arrive at 14 for the final component?  (This OID is probably fine, I just want a better idea of what's going on with this arc.)